components/UserListDisplay.tsx

```diff
@@ -16,12 +16,48 @@ import { CopyToClipboard } from "./ui/CopyToClipboard"
 import { InfoButton } from "./ui/InfoButton"
 import { MultiselectUsers } from "./ui/MultiselectEmail"
 
+function ListContent({ 
+    bigHeading,
+    compact,
+    userList
+  }: {
+    bigHeading: boolean;
+    compact: boolean;
+    userList: UserListWithAuthorAndUsers;
+  }) {
+    return (
+      <>
+        {bigHeading ? (
+          <h3 className="text-xl font-semibold my-0">{userList.name}</h3>
+        ) : (
+          <>
+            {userList.name}
+            {!compact && (
+              <ArrowTopRightOnSquareIcon className="inline ml-1 h-3 w-3 text-neutral-600" />
+            )}
+
+            {(userList.users.length > 0 || userList.emailDomains.length === 0) && (
+              <span className="block text-xs my-auto font-normal text-neutral-400 text-left">
+                {userList.users.length} member{userList.users.length === 1 ? "" : "s"}
+              </span>
+            )}
+            {userList.emailDomains.length > 0 && (
+              <span className="block text-xs my-auto font-normal text-neutral-400">
+                {userList.emailDomains.map((domain) => `anyone@${domain}`).join(", ")}
+              </span>
+            )}
+          </>
+        )}
+      </>
+    );
```

Comment:
nit: Prettier should remove semicolons, I think it's worth setting it up in your editor to format on save/paste/commit/etc. You can then just forget about formatting (saves a lot of keystrokes!)

https://prettier.io/docs/en/editors